api/apps/api/src/modules/projects/projects-crud.service.ts

```diff
@@ -272,9 +294,32 @@ export class ProjectsCrudService extends AppBaseService<
       );
     }
 
+    query
+      .andWhere(`acl.user_id = :userId`, {
+        userId: info?.authenticatedUser?.id,
+      })
+      .andWhere(`acl.role_id = :roleId`, {
+        roleId: Roles.project_owner,
+      });
```

Comment:
I know this is what is asked in the story, but looking at the implementation, I wonder if we should drop the check of role and just list every project the user has a role on, as we won't be setting other roles right now (so in practice even without this check the end result will be the same as in the story), with the tiny benefit of not having to remove this when we add full ACL checks (although in any case we'll have to revisit a lot of things by then)... what do you think @kgajowy ?

Reply:
Once we will want to add full ACL (or RBAC) checks, it will require a lot more changes I believe.

Either change this one and update documentation/requirements, or let it be. Up to you @hotzevzl

Reply:
yep, let's leave it as is.